projects/goharbor/harbor/Makefile

```diff
@@ -42,8 +42,8 @@ RELEASEVERSION=$(shell cat $(REPO)/VERSION)
 
 IMAGE_NAMES?=harbor-db harbor-portal harbor-core harbor-log harbor-nginx harbor-jobservice harbor-registry harbor-registryctl harbor-redis harbor-exporter harbor-trivy
 
-BINARY_TARGET_FILES=harbor-core harbor-jobservice harbor-registryctl harbor-migrate-patch harbor-migrate harbor-exporter
```

Comment:
They [removed the `migrate-patch` utility](https://github.com/goharbor/harbor/pull/20619) in version `v2.11.1`. So we can stop building it.